src/visitors/pages/qrcode.js

```diff
@@ -101,6 +99,10 @@ export default class QRCode extends Component {
 
         Visitors.search({ qrCode: content })
           .then((res) => {
+            if (!res.data.result) {
+              throw new Error('No user found');
```

Comment:
User gets no feedback if we throw.

Reply:
Falls into the catch, which is the qr error page. I don’t think we need any additional feedback (at this point)

Reply:
Falls into the catch, which forward to the qr error page. I don’t think we need any additional feedback (at the moment)